test/runtests.jl

```diff
@@ -1,6 +1,10 @@
 using SimTreeUtils
 using Test
+using BSON
 
+include("testutils.jl")
 @testset "SimTreeUtils.jl" begin
     # Write your tests here.
+    @test typeof(SimTreeUtils.stsimulate(testmainfunction))== Dict{String,Any}
```

Comment:
I don't get what this actually tests. reading a bson like a Dict is a functionality provided by the BSON.jl package and not this one. So no need to test that.
New test should target the new funcionality introduced which is the `stsimulate` funciton.

Reply:
This test checks if the function ran and returns the dict.
The other function checks if the saving worked.